### ISSUE TITLE
Better quick start guide?

### ISSUE THREAD

**Issue by shibekin69** [ncannasse/castle#19]
Hi!

I'm trying to get this to run with HaxeFlixel (is this compatible by the way?) by adding it into my project's project.xml file under the <haxelib> tag. I'm able to add in Data.hx with the macro, but when I try using Data in my playState by importing it and trying to use the object, I'm seeing a null object. Well, how do you retrieve the data from a cdb file then? I can't get the test hxproj to run either since I'm running into an error about g being a duplicate class. Haxe is at 3.2.0.

**Comment by ncannasse** [comment]
As stated in the documentation, you need to use Data.load to load your CDB content.
One possibility for that is to use HaxeFlixel resource manager, or Haxe resources by compiling with -resource myfile.cdb then haxe.Resource.getString('myfile.cdb') at runtime

**Comment by ncannasse** [comment]
PS : i have updated test

**Comment by shibekin69** [comment]
Hi Nicolas, thanks for the tip! It wasn't apparent how I should be adding that -resource xxx.cdb parameter in FlashDevelop (I used Flixel to generate a template for me to work with), but this example by prog4mr solved the trick:

https://gist.github.com/ProG4mr/52e917037930ed7487e5

//In project.xml
<haxeflag name="-resource" value="assets/data/test.cdb@test.cdb" />

**Comment by awreese** [comment]
I also found the Haxe integration instructions to be a bit lacking.  I'm using HaxeDevelop and compiling to HTML5 (lime/javascript), so all the instructions targeting Flash are a bit outdated.  Following this [GamesFromScratch basic tutorial](http://www.gamefromscratch.com/post/2015/11/14/CastleDB-A-Game-Database-A-Level-Editor-Its-Both.aspx) got me close, but @shibekin69 's [post](https://github.com/ncannasse/castle/issues/19#issuecomment-105415024) finally got it working for me.

For Haxe/HaxeFlixel => HTML5 in the Project.XML file, under the Libraries sections I added these entries:
```
    <haxelib name="castle"/>
    <haxeflag name="-resource" value="assets/data/myDB.cdb" />
```

Steps:
1. HaxeDevelop: Project->New Project, select 'HaxeFlixel Project'
1. Create usual data.hx file to setup the cdb linking
1. Add above entries to 'Project.XML'
1. Instantiate your database file in Main.hx::new()
